scripts/import/import_all_sv_data.pl

```diff
@@ -1487,65 +1524,88 @@ sub parse_9th_col {
     if ($key eq 'phenotype') {
       $value =~ s/, /__/g;
       foreach my $phe (split(',', $value)) {
+
+        $phe = decode_text($phe);
+
         $phe =~ s/__/, /g;
         $info->{phenotype}{$phe} = 1;
       }
     }
     $info->{phenotype}{$value} = 1 if ($key eq 'phenotype_description');
     $info->{phenotype_link} = $value if ($key eq 'phenotype_link' || $key eq 'phenotype_id');
+
+    # Allele count and frequency
+    if ($key eq 'allele_count') {
+      $info->{allele_count} = $value;
+    }
+
+    if ($key eq 'allele_frequency') {
+      $info->{allele_frequency} = $value;
+    }
+
+    if ($key eq 'allele_number') {
+      $info->{allele_number} = $value;
+    }
+
   }
-  
-  
+
+
   ## Phenotype(s) ##
-  
+
   if ($info->{phenotype_link}) {
-    
+
     my @phenotype_links = split(',',$info->{phenotype_link});
 
     foreach my $p_link (@phenotype_links) {
+
       # Look at the MedGen data
       if ($medgen_file && $p_link =~ /^MedGen:\w+$/i) {
         $p_link =~ /^MedGen:(\w+)$/i;
         my $phenotype_value = get_medgen_phenotype($1);
         $info->{phenotype}{$phenotype_value} = 1 if (defined($phenotype_value) && $phenotype_value ne '');
       }
-    
+
       # Look at the HP ontology data
       if ($hpo_file && $p_link =~ /^HP:\d+$/) {
         my $phenotype_value = get_hpo_phenotype($p_link);
         $info->{phenotype}{$phenotype_value} = 1 if (defined($phenotype_value) && $phenotype_value ne '');
       }
     }
   }
-  
+
   # Flag to know if the entry is a sv or a ssv
   $info->{is_ssv} = ($info->{ID} =~ /ssv/) ? 1 : 0;
-  
+
   # Somatic alias
   if ($info->{is_somatic} == 1 && $info->{alias} =~ /^(.+)_\d+$/) {
     $info->{alias} = $1;
   }
-  
+
   # Check if a SV has its SSV(s) labelled as 'somatic'
   if ($somatic_study && $info->{is_ssv} == 0) {
     my $cmd  = "grep 'parent=".$info->{ID}."' $fname";
     my $text = `$cmd`;
     $info->{is_somatic} = 1 if ($text =~ /var_origin=Somatic/i || !$text);
   }
-  
+
   return $info;
 }
 
-
+# Replace weird characters
+# dbVar weird characters are still in the files - replace them here
```

Comment:
Can that not also go into replace_char?

Reply:
I didn't think about putting there because they are very specific to dbVar files. I don't think they were found anywhere else but I'll put it there to be more consistent.